README.md

```diff
@@ -282,11 +318,22 @@ Advanced examples of using this trace event mechanism are:
     cloning objects to make them immutable is relatively expensive and may generate significant
     overhead.
 
-## More information
+### More information on trace events
 
 For more information, for example on how to define and register trace event consumers, please refer
 to the documentation in the `Tracer` class.
 
+## License
+
+Copyright (c) 2020 - 2020 TomTom N.V. All rights reserved.
+
+This software is the proprietary copyright of TomTom N.V. and its subsidiaries and may be
+used for internal evaluation purposes or commercial use strictly subject to separate
+licensee agreement between you and TomTom. If you are the licensee, you are only permitted
+to use this Software in accordance with the terms of your license agreement. If you are
+not the licensee then you are not authorised to use this software in any manner and should
+immediately return it to TomTom N.V.
```

Comment:
Does this license apply to open source projects as well? _"you are only permitted to use this Software in accordance with the terms of your license agreement"_ doesn't sound applicable to something that can be used without an explicit license agreement with TomTom. (Though I'm no legal expert so I may be wrong. :) )

Reply:
Good catch.